src/plugin/cilium.go

```diff
@@ -253,6 +253,97 @@ func ConvertCiliumFlowToKnoxNetworkLog(ciliumFlow *cilium.Flow) (types.KnoxNetwo
 	return log, true
 }
 
+func ConvertSQLiteCiliumLogsToKnoxNetworkLogs(docs []map[string]interface{}) []types.KnoxNetworkLog {
```

Comment:
We do not use this function anywhere I believe. Can we remove this function?

Reply:
It is used in cilium.go file itself
```
func ConvertCiliumNetworkLogsToKnoxNetworkLogs(dbDriver string, docs []map[string]interface{}) []types.KnoxNetworkLog {
	if dbDriver == "mysql" {
		return ConvertMySQLCiliumLogsToKnoxNetworkLogs(docs)
	} else if dbDriver == "sqlite3" {
                return ConvertSQLiteCiliumLogsToKnoxNetworkLogs(docs)
        } else if dbDriver == "mongo" {
		return ConvertMongodCiliumLogsToKnoxNetworkLogs(docs)
	} else {
		return []types.KnoxNetworkLog{}
	}
}
```